frontend/src/modules/_shared/DataProviderFramework/settings/implementations/InputNumberSetting.tsx

```diff
@@ -9,20 +9,22 @@ import type { SettingCategory } from "../settingsDefinitions";
 
 type ValueType = number | null;
 
+type StaticProps = { min?: number; max?: number };
+
 export class InputNumberSetting implements CustomSettingImplementation<ValueType, SettingCategory.NUMBER> {
-    private _staticMinMax: { min: number; max: number } | null;
+    private staticProps: StaticProps | null;
```

Comment:
For consistency with `DropdownStringSetting`: make attribute have suffix "_", i.e. `_staticProps`